contracts/test/v0.8/automation/helpers.ts

```diff
@@ -7,12 +7,17 @@ import { AutomationRegistryLogicB2_2__factory as AutomationRegistryLogicBFactory
 import { IAutomationRegistryMaster as IAutomationRegistry } from '../../../typechain/IAutomationRegistryMaster'
 import { IAutomationRegistryMaster__factory as IAutomationRegistryMasterFactory } from '../../../typechain/factories/IAutomationRegistryMaster__factory'
 
+const ZERO = 0
```

Comment:
i had to add these bc SonarQube thinks using magic numbers 0, 1, 2 directly causes brain overload